tests/gen_cert_test.py

```diff
@@ -105,3 +111,16 @@ def test_cert_upload():
     r = urllib2.urlopen(download_url)
     with tempfile.NamedTemporaryFile(delete=True) as f:
         f.write(r.read())
+
+
+def test_render_flair_function():
+    """Make sure flair rendering function behaves properly"""
```

Comment:
Despite how it's done elsewhere, we should always use the multiline docstring format.